src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/InstalledAndTransitivePackageFeed.cs

```diff
@@ -28,12 +28,17 @@ public override async Task<SearchResult<IPackageSearchMetadata>> ContinueSearchA
             var searchToken = continuationToken as FeedSearchContinuationToken ?? throw new InvalidOperationException(Strings.Exception_InvalidContinuationToken);
             cancellationToken.ThrowIfCancellationRequested();
 
+            PackageCollectionItem[] matchesInstalled = PerformLookup(_installedPackages.GetLatestCore(), searchToken);
+
             // Remove transitive packages from project references
```

Comment:
Is this comment accurate? It doesn't seem to be removing transitive packages from anything, but it does seem to be creating a collection that has only transitive packages. Am I mistaken?